vizro-ai/docs/pages/user-guides/add-generated-chart-usecase.md

```diff
@@ -1,6 +1,8 @@
 # How to add Vizro-AI created charts into a Vizro dashboard
 
-This guide explains how to add a chart generated by Vizro-AI to your [Vizro](https://github.com/mckinsey/vizro/tree/main/vizro-core) dashboard.
+This guide explains different ways in which you can add a chart generated by Vizro-AI to your [Vizro](https://github.com/mckinsey/vizro/tree/main/vizro-core) dashboard.
+
+###  Access chart-generating code
```

Comment:
```suggestion
##  Access chart-generating code
```

Reply:
I went back and forth with the wording here but I think the elements are that you are using the code that Vizro-AI generates, so it's best to explain that simply. It means the word "Chart" is omitted. An alternative would be "Use generated code to create the chart" or "Use Vizro-AI generated code to create the chart" but that gets quite wordy. Which do you prefer?

Reply:
I actually prefer the committed version "Use Vizro-AI's generated code". Its simple and it is obvious from the introduction above that we are trying to generate a chart using VizroAI. Or do you prefer the examples with the word chart in?

Reply:
I don't think we need the word chart here, let's go with your preference.